### PR TITLE
Feature/add pagination to admin page

### PR DESCRIPTION
**Pull request by bagiratania98**
T - https://trello.com/c/K1sVPxQ4/71-add-pagination-to-admin-pages
Add gem pagy 
Set up pagination and records per page
Add pagination to all admin pages

### CODE REVIEW

File: app/controllers/admins/users_controller.rb [hunk 1]
```diff
@@ -6,11 +6,7 @@ class UsersController < AdminsController
     helper_method :last_super_admin?
 
     def index
-      @users = if params[:sort] && User.column_names.include?(params[:sort])
-                 User.order(params[:sort])
-               else
-                 User.all
-               end
+      @pagy, @users = pagy(User.all)
```
Comment (mmrshk) [inline app/controllers/admins/users_controller.rb:9]: what happened with sorting?

File: config/initializers/pagy.rb [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+# than the default pagy internal i18n (see above)
+# See https://ddnexus.github.io/pagy/extras/i18n
+require 'pagy/extras/i18n'
+
+# Default i18n key
+# Pagy::DEFAULT[:i18n_key] = 'pagy.item_name'   # default
+
+
+# When you are done setting your own default freeze it, so it will not get changed accidentally
+Pagy::DEFAULT.freeze
```
Comment (mmrshk) [inline config/initializers/pagy.rb:244]: EOF

**Review by mmrshk — APPROVED** [review]
LGTM! Please, ask Misha for review. And please, show me the result via demo.

File: config/initializers/pagy.rb (hunk 2 above)
Comment (bagiratania98) [reply config/initializers/pagy.rb:244]: Sorry)

File: app/assets/stylesheets/admin.scss [hunk 3]
```diff
@@ -193,23 +193,21 @@ select {
...
-
-    .label-records-per-page  {
+    label {
       margin-left: 5px;
       color: $blue-filter-color;
       font-size: 20px;
     }
 
     input[type=number] {
+      padding-right: 6px !important;
```
Comment (MishaKlymenko) [inline app/assets/stylesheets/admin.scss:203]: is it possible to assign a rule without !important

File: app/assets/stylesheets/admin.scss [hunk 4]
```diff
@@ -193,23 +193,21 @@ select {
...
+      color: $white-color;
+      width: 50px !important;
```
Comment (MishaKlymenko) [inline app/assets/stylesheets/admin.scss:205]: is it possible to assign a rule without !important

File: app/assets/stylesheets/admin.scss [hunk 5]
```diff
@@ -193,23 +193,21 @@ select {
...
+      height: 40px;
+      background-color: $blue-filter-color;
+      border-radius: 10px;
+      border: none !important;
```
Comment (MishaKlymenko) [inline app/assets/stylesheets/admin.scss:209]: is it possible to assign a rule without !important

**Review by MishaKlymenko — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/admin.scss (hunk 3 above)
Comment (bagiratania98) [reply app/assets/stylesheets/admin.scss:216]: maybe no

File: app/assets/stylesheets/admin.scss (hunk 4 above)
Comment (bagiratania98) [reply app/assets/stylesheets/admin.scss:218]: no)

File: app/assets/stylesheets/admin.scss (hunk 5 above)
Comment (bagiratania98) [reply app/assets/stylesheets/admin.scss:222]: no)

**Pull request merged**